composer.json

```diff
@@ -156,6 +156,7 @@
   },
   "conflict": {
     "symfony/dependency-injection": "3.4.15|3.4.16",
+    "symfony/monolog-bundle": "3.4.0",
```

Comment:
```suggestion
    "symfony/monolog-bundle": ">=3.4.0",
```

Reply:
why? but ok

Reply:
Because we'd still have problems when symfony/monolog-bundle releases v3.4.1. It's safer to use `>=` when adding the conflict and change it to a closed interval (or a single version) when we know it's been fixed.